spec/features/sign_up_as_an_organisation_spec.rb

```diff
@@ -75,7 +75,7 @@
     it_behaves_like 'errors in form'
 
     it 'tells the user that the passwords do not match' do
-      expect(page).to have_content 'Password must match confirmation'
+      expect(page).to have_content "Password confirmation doesn't match Password"
```

Comment:
The previous text feels more "human" to me - is it easy to tweak the devise translations .yml to change the message?